README.md

```diff
@@ -87,8 +87,15 @@ spec:
 ## Quick Start
 
 Clone the repository and run `make local` in an existing `kube:admin` OpenShift
-CLI session. 
+CLI session
 
+### Running operator locally
+
+Clone the repository and run `make local` in an existing `kube:admin` OpenShift CLI session.
+
+### Installing the operator in a cluster
+
+Follow steps from the [Installation Guide](docs/installation.md) to install the Service Binding Operator in OpenShift 4.5+ or any other Kubernetes cluster.
```

Comment:
We need a section somewhere about usage of `install.sh` script.

Reply:
Done